src/Configuration/DicomWebConfiguration.cs

```diff
@@ -28,6 +28,28 @@ public class DicomWebConfiguration
         [ConfigurationKeyName("maximumNumberOfConnections")]
         public int MaximumNumberOfConnection { get; set; } = 2;
 
+        /// <summary>
+        /// Gets or set the maximum allowed file size in bytes with default to 2GiB.
+        /// </summary>
+        [ConfigurationKeyName("maxAllowedFileSize")]
+        public long MaxAllowedFileSize { get; set; } = 2147483648;
+
+        /// <summary>
+        /// Gets or set the maximum memory buffer size in bytes with default to 30MiB.
+        /// </summary>
+        [ConfigurationKeyName("memoryThreshold")]
+        public int MemoryThreshold { get; set; } = 31457280;
+
+        /// <summary>
+        /// Timeout, in seconds, to wait for instances before notifying other subsystems of data arrival
+        /// for the specified data group.
+        /// Defaults to two seconds.
+        /// Note: the currently implementation of DICOMweb expects the entire payload to be received in a
+        /// single POST request, therefore, the timeout value may be insignificant unless the load of the
+        /// network affects the upload speed.
+        /// </summary>
+        public uint Timeout { get; set; } = 2;
```

Comment:
Looks this is because the backend is using the same triggering mechanism as the DIMSE Store SCP, kind of defeat the purpose of requesting all applicable instances be in one STOW-RS request, as this is not using the request processing completion as trigger.

Reply:
Right; but we can simply increase this value to enable multiple POSTs.